From b0603bea7ab3c1b0685d14026d29c8e7cf84b757 Mon Sep 17 00:00:00 2001
From: Nadav Hugim <nhugim@habana.ai>
Date: Tue, 22 Feb 2022 13:21:07 +0200
Subject: [PATCH] [SW-68159] post_build: fix preboot image create

there was an error in preboot file name in the scrip.
we used preboot.bin but need to use gaudi-preboot.bin.le

Change-Id: If299c7ba3e8bc7de40dd2477a767522d92147207
Signed-off-by: Nadav Hugim <nhugim@habana.ai>
---

diff --git a/board/habanalabs/common/post-image-images-pointers.sh b/board/habanalabs/common/post-image-images-pointers.sh
index 3ddab43..66e576c 100755
--- a/board/habanalabs/common/post-image-images-pointers.sh
+++ b/board/habanalabs/common/post-image-images-pointers.sh
@@ -7,7 +7,7 @@
 HL_BASE_PATH=${BR2_EXTERNAL_HABANALABS_PATH}/board/habanalabs
 COMMON_BASE=${HL_BASE_PATH}/common
 BOARD_BASE=${HL_BASE_PATH}/${HL_DEV}
-PREBOOT_BINFILE=${PREBOOT_BINFILE:-preboot.bin}
+PREBOOT_BINFILE=${PREBOOT_BINFILE:-${HL_DEV}-preboot.bin.be}
 SPEC_BOOT_DESC_FILE=${BR2_EXTERNAL_HABANALABS_PATH}/../specs/${HL_DEV}/${HL_DEV}_boot_info.h
 
 source ${COMMON_BASE}/image-utility.sh
@@ -74,7 +74,7 @@
 	printf "%08x" $img_loadl | rev_bytes_stdin
 
 	# Prepare checksum
-	local preboot_file=`echo ${HL_DEV}-${PREBOOT_BINFILE} | cut -f1 -d"."`
+	local preboot_file=`echo ${PREBOOT_BINFILE} | cut -f1 -d"."`
 	local preboot_le="${BINARIES_DIR}/${preboot_file}.bin.le"
 	local PRE_CKSUM=`${SHA_CHKSUM} ${preboot_le} | cut -d' ' -f1`
 	hl_chksum_add ${PRE_CKSUM}
